cli/dcoscli/cluster/main.py

```diff
@@ -368,10 +369,10 @@ def setup(dcos_url,
     :rtype: int
     """
 
-    with cluster.setup_directory() as temp_path:
+    with util.tempdir() as tempdir:
```

Comment:
Is the `cluster.setup_directory()` function still in use after this change? It can probably be removed.

Reply:
Still used in https://github.com/dcos/dcos-cli/blob/a705fa3587cb5ba873aede859a9aa42c37f5f40a/tests/test_cluster.py#L117 but it's the test failing in CI thus I'm gonna update it now!

Reply:
If it's now only used in a test we should remove that function and either : 
- update the test accordingly
- remove the test (if you think it's not needed anymore)